### ISSUE TITLE
data augmentation in pytorch

### ISSUE THREAD

**Issue by Sirius083** [Lyken17/SparseNet#8]
Can you post your pytorch implementation of data augmentation/training details?
(Is it same with that in tensorflow?)
Since the data augmentation is per-channel-mean while the tensorflow implementation is per-pixel-mean
Thanks in advance

**Comment by Lyken17** [comment]
I don't think these two will make much difference.

**Comment by Lyken17** [comment]
Let me know if this detail makes a significant change in performance. If that is true, I will re-rewrite the data pre-processing after nips.

**Comment by Sirius083** [comment]
![tmp](https://user-images.githubusercontent.com/22949166/57666507-be9e5280-7632-11e9-88d9-ae31322f8176.png)
This is the results I got under tensorflow implementation of sparseNet under densenet preprocessing implementation. I didnot get same accuracy as as densenet as well(1% accuracy lower), maybe this is due to tensorflow and pytorch optimization difference. 
I will have a further look into that
Thank you very much

**Comment by Sirius083** [comment]
> I don't think these two will make much difference.

Thanks for your reply, I finnaly found it is because tensorflow implementation did not include the L2 normal of batch normalization parameters weight decay in tensorpack implementaion. After that, the model give better results.
